src/configure/configure.ts

```diff
@@ -405,16 +405,20 @@ class Orchestrator {
                 };
             }
             else {
-                let repositoryProvider: string = "Other";
+                let repositoryProvider: string;
 
                 if (remoteUrl.indexOf("bitbucket.org") >= 0) {
                     repositoryProvider = "Bitbucket";
                 }
                 else if (remoteUrl.indexOf("gitlab.com") >= 0) {
                     repositoryProvider = "GitLab";
                 }
+                else {
+                    repositoryProvider = remoteUrl;
+                }
 
                 telemetryHelper.setTelemetry(TelemetryKeys.RepoProvider, repositoryProvider);
+                telemetryHelper.setTelemetry(TelemetryKeys.IsErrorWhitelisted, "true");
```

Comment:
You should also put a if condition for IsErrorWhitelisted on line 66 and don't set the result to failed.

Reply:
So basically have a variable IsErrorWhitelisted and put condition according to that?
(I though our query for results will have the condition based on the telemetry key)

Reply:
Yes have a variable and put failed/succeed condition acc to that and also set the whitelisted telemetry values. So by default we will get all the whitelisted error on succeeded and if we need to dig down further in future we can use the isWhitelisted flag in kusto.

Reply:
updated